### DISCUSSION TITLE
accept doesn't return anything

### DISCUSSION THREAD

**Discussion by seohyeon0322** [ANLAB-KAIST/KENSv3#38] [Project Q&A 1. Application Layer (Socket)]
I tested using echo-non-kens.
I checked that bind and listen succeeded. However, accept() doesn't return anything, even -1. 

I put second parameter client_address in type of struct sockaddr_in and casted it to struct sockaddr *. Also, I initialized client_address 's size , and put it into third parameter. However, accept() returns anything when I test   `./echo-non-kens s 0.0.0.0 9000 hello-client `.

**Comment by Pusnow** [comment]
It seems your server is blocked and waiting a new connection. Open another terminal window and run `./echo-non-kens c 127.0.0.1 9000 hello`
